README.md

```diff
@@ -38,10 +157,10 @@ The rendering in the function will be tracked and automatically re-rendered when
 This can come in handy when needing to pass render function to external components (for example the React Native listview), or if you want to observe only relevant parts of the output for a performance reasons.
 
 ```jsx
-import { Observer, useObservable } from "mobx-react-lite"
+import { Observer, useLocalStore } from "mobx-react-lite"
 
 function ObservePerson(props) {
-    const person = useObservable({ name: "John" })
+    const person = useLocalStore(() => ({ name: "John" }))
```

Comment:
Based on this little code I think people might be fairly confused why not to use `useAsObservableSource`. It would be simpler in this case imo.

Reply:
I don't think so, because it is supposed to be mutable.
5 lines below:
```tsx
    <button onClick={() => (person.name = "Mike")}>No! I am Mike</button>
```
so `useAsObservableSource` would not work here.
I think it is fine like this